karma.conf.js

```diff
@@ -1,4 +1,75 @@
-// This allows the karma conf to use ES6.
-require('./server.babel');
+// Karma configuration
+/* eslint-disable no-console */
+require('babel-register');
+const webpackDevConfig = require('./src/config/webpack.dev.config.babel').default;
```

Comment:
This could drop the `./src/` bit, I think. That seems to be how we normally require stuff.